lib/services/profiles-api-service.js

```diff
@@ -61,6 +63,18 @@ function profileApiServiceFactory(
         return _.flattenDeep([macs]);
     };
 
+    ProfileApiService.prototype.setNode = function(query) {
```

Comment:
This function name doesn't really make sense to me? Since you're setting Mac and Ip and not node Id?